src/main/menu/templates/help.js

```diff
@@ -76,12 +76,12 @@ export default function () {
         shell.openExternal('https://github.com/marktext/marktext')
       }
     }, {
-      label: 'Follow @Jocs on Github',
+      label: 'Follow on Github',
       click () {
         shell.openExternal('https://github.com/Jocs')
       }
     }, {
-      label: 'Follow us on Twitter',
+      label: 'Follow on Twitter',
```

Comment:
Personally I think the old string was better, as your suggestion doesn’t really say who you’re following.

Reply:
Mostly I wanted the Twitter and Github entries to have the same structure. I liked the "Follow on *website*" form because it is simple and (to me) entirely clear. (Who else would be the target but the application developers?) No choice is wrong, but parallel structure is always best.

Reply:
A alternative might be to use “Follow us on {website}”; the structure is unified and it (to me) reads better.

Reply:
By "alternative" are you referring to the original form?

Reply:
To me it reads better intrinsically without the extra word, but  "follow us on twitter" seems to have become the common usage. But how does it unify structurally with the Github one?

Reply:
So "Follow us on Twitter" and "Follow us on Github"?

Reply:
I’d say lowercase “us”.